fastq_filtrator.py

```diff
@@ -0,0 +1,96 @@
+
+def fun_gc_bounds(gc_bounds, GC_content):
+    left_border = 0.0
+    rigth_border = 100.0
+    if isinstance(gc_bounds, float) | isinstance(gc_bounds, int) == True:

```

Comment:
`== True` не имеет особого смысла, так как приоритет операций будет вот такой
 ```
if ( isinstance(gc_bounds, float) | ( isinstance(gc_bounds, int) == True ) ):
```
И ещё в `if`, как правило, не принято использовать |, & и ~. Вместо них лучше `or`, `and` и `not` 
```suggestion
    if isinstance(gc_bounds, float) or isinstance(gc_bounds, int):

```

Reply:
Хорошо, что не забыла про `float` 👍